### PR TITLE
Feat: List of places

### PR DESCRIPTION
**Pull request by rpietrek**

### CODE REVIEW

File: src/app/places/places.component.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+interface Place {
+  Id: number;
+  Name: String;
+  Address: String;
+  Phone: String;
+  Link: String;
+  Max_persons: String;
+  Tag: String;
+  Description: String;
+}
```
Comment (Sebastian-plDev) [inline src/app/places/places.component.ts:13]: in my opinion any interface should be placed in separate file in models folder

File: src/app/pages.service.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import data from '../app/data/places-katowice.json';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PagesService {
+  places: any = data;
```
Comment (Sebastian-plDev) [inline src/app/pages.service.ts:8]: why :any? beter :Place[]

File: src/app/pages.service.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  constructor() {}
+  getPlaces(): any {
```
Comment (Sebastian-plDev) [inline src/app/pages.service.ts:11]: the same as above

File: src/app/places/places.component.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+@Component({
+  selector: 'app-places',
+  templateUrl: './places.component.html',
+  styleUrls: ['./places.component.scss'],
+})
+export class PlacesComponent implements OnInit {
+  PlacesFromService: any = null;
```
Comment (Sebastian-plDev) [inline src/app/places/places.component.ts:21]: :Place[]

File: src/app/places/places.component.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  constructor(private pagesService: PagesService) {}
+
+  ngOnInit(): void {
+    this.fetchPlaces();
+  }
+
+  private fetchPlaces(): any {
```
Comment (Sebastian-plDev) [inline src/app/places/places.component.ts:28]: this should be void it does not return value

File: src/app/pages.service.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
```
Comment (Sebastian-plDev) [inline src/app/pages.service.ts:1]: I think there is a typo in the name of this service - the service which is responsible for returning places should be called PlacesService

File: src/app/places/places.component.ts (hunk 1 above)
Comment (rpietrek) [reply src/app/places/places.component.ts:13]: I make a folder called models and file in this folder called place.

**Review by Sebastian-plDev — APPROVED** [review]

**Pull request merged**